runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowWorkerLoggingOptions.java

```diff
@@ -38,7 +38,47 @@
    * The set of log levels that can be used on the Dataflow worker.
    */
   public enum Level {
-    DEBUG, ERROR, INFO, TRACE, WARN
+    /**
+     * Special level used to turn off logging.
+     *
+     * @see java.util.logging.Level#OFF
```

Comment:
It still seems useful to know, since the user may have setup some JUL handling in a config file or something. We can change the docs if we change how these relate to JUL...